.pypirc

```diff
@@ -0,0 +1,8 @@
+[distutils]
+index-servers =
+    pypi
+
+[pypi]
+  repository = https://upload.pypi.org/legacy/
+  username = __token__
+  password = #PYPI_TOKEN
```

Comment:
this file is a template - not something that should be put in the root of the project.
think about a developer who forks opal - this can interfere with his/her `.pypirc`.

You should instead generate this file inside the action from a template string.
it will be generated only on the runner machine.